Makefile

```diff
@@ -36,7 +36,9 @@ endif
 ifdef TESTING_ENABLE
     OPENMLDB_CMAKE_FLAGS += -DTESTING_ENABLE=$(TESTING_ENABLE)
 endif
-
```

Comment:
Do we need to support MAC_TABLET_ENABLE by default.
Otherwise, we should notify users/developers that the make in MacOS won't build a tablet.

Reply:
I don't know why `MAC_TABLET_ENABLE ` is false by default. Do you know it?

Reply:
+1 for making MAC_TABLET_ENABLE as true by default.

It doesn't increase the output binary size and meaningless for MacOS users.

Reply:
> 

Because we didn't support building a mac tablet at the very beginning. Mac tablet is experimental for a long time.

Reply:
But we released the macos version of openmldb(contains servers), that means we treat it official.
Let's set `MAC_TABLET_ENABLE ` as true by default.